microservices/order/tests/test_queries/test_services.py

```diff
@@ -98,6 +113,51 @@ async def asyncSetUp(self) -> None:
     async def asyncTearDown(self) -> None:
         await self.injector.unwire()
 
+    async def test_get_orders(self):
+        now = datetime.now(tz=timezone.utc)
+
+        payment_detail = {
+            "card_holder": "John",
+            "card_number": 2424242424242424,
+            "card_expire": "12/24",
+            "card_cvc": "123",
+        }
+
+        shipment_detail = {
+            "name": "Jack",
+            "last_name": "Johnson",
+            "email": "jack@gmail.com",
+            "address": "Calle Gran Víia 34",
+            "country": "Spain",
+            "city": "Madrid",
+            "province": "Madrid",
+            "zip": 34324,
+        }
+
+        expected = await gather(
+            Order.create(
+                entries=EntitySet(),
+                payment_detail=payment_detail,
+                shipment_detail=shipment_detail,
+                status=OrderStatus.CREATED,
+                customer=uuid4(),
+            ),
+            Order.create(
+                entries=EntitySet(),
+                payment_detail=payment_detail,
+                shipment_detail=shipment_detail,
+                status=OrderStatus.CREATED,
+                customer=uuid4(),
+            ),
+        )
+
+        request = _FakeRequest({"uuids": [v.uuid for v in expected]})
+
+        response = await self.service.get_orders(request)
+        observed = await response.content()
+
+        self.assertEqual(expected, observed)
+
```

Comment:
As this is an snapshot test I think it's better to remove it from the query service tests because it's an internal implementation provided by the `minos` framework.